wp-includes/widgets/class-wp-widget-image.php

```diff
@@ -29,9 +29,16 @@ public function __construct() {
 		) );
 
 		$this->l10n = array_merge( $this->l10n, array(
-			'no_media_selected' => __( 'No image selected' ),
-			'edit_media' => __( 'Edit Image' ),
 			'change_media' => __( 'Change Image' ),
+			'edit_media' => __( 'Edit Image' ),
+			'missing_attachment' => sprintf(
+				/* translators: placeholder is URL to media library */
+				__( 'We can&#8217;t find that image. Check your <a href="%s">media library</a> and make sure it wasn&#8217;t deleted.' ),
+				esc_url( admin_url( 'upload.php' ) )
+			),
+			/* translators: %s is widget count */
+			'media_library_state' => _n_noop( 'Image Widget', 'Image Widget (%s)' ),
```

Comment:
Maybe these should actually be two separate strings. 

For more about this, see https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/567#issuecomment-224788685